auslib/blobs/schemas/apprelease-v1.yml

```diff
@@ -22,6 +22,9 @@
       description: Blob identifier number
       enum:
         - 1
+    has_wnp:
```

Comment:
This isn't necessary. Remember: has_wnp is not part of the blob schema itself, just part of the JSON response from the web API.